csrc/mmdeploy/device/cpu/cpu_device.cpp

```diff
@@ -47,11 +50,12 @@ class CpuHostMemory : public NonCopyable {
     size_ = 0;
   }
   size_t size() const { return size_; }
-  void* data() const { return data_; }
+  void* data() const { return owned_data_ ? aligned_data_ : data_; }
 
  private:
   size_t size_;
   void* data_;
+  void* aligned_data_{nullptr};
```

Comment:
Unaligned data almost useless for deep learning inference, how about remove `data_` and keep it aligned by default.

Reply:
CpuHostMemory accepts to build from raw ptr.
https://github.com/open-mmlab/mmdeploy/blob/e28e47128d94315dc1099562925d9da992d47a72/csrc/mmdeploy/device/cpu/cpu_device.cpp#L35-L40

The data is not owned by the class and we can not guarantee that the data is aligned.